File: CS/T548916/Models/EmployeeTask.cs
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;
using System;

namespace dxSample.Models
{
    [JsonConverter(typeof(StringEnumConverter))]
    public enum Priority { High, Normal, Low, Urgent }
    public class EmployeeTask
    {
        public int Completion { get; set; }
        public DateTime DueDate { get; set; }
        public int ID { get; set; }
        public Priority Priority { get; set; }
        public DateTime StartDate { get; set; }
        public string Status { get; set; }
        public string Subject { get; set; }
    }
}

File: VB/T548916/Models/EmployeeTask.vb
Imports Newtonsoft.Json
Imports Newtonsoft.Json.Converters
Imports System

Namespace T548916.Models
	<JsonConverter(GetType(StringEnumConverter))>
	Public Enum Priority
		High
		Normal
		Low
		Urgent
	End Enum
	Public Class EmployeeTask
		Public Property Completion() As Integer
		Public Property DueDate() As Date
		Public Property ID() As Integer
		Public Property Priority() As Priority
		Public Property StartDate() As Date
		Public Property Status() As String
		Public Property Subject() As String
	End Class
End Namespace